src/v3/webpack.release.config.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { resolve } from 'path';
+import { readFileSync } from 'fs';
+
+import webpack from 'webpack';
+import { merge } from 'webpack-merge';
+import MiniCssExtractPlugin from 'mini-css-extract-plugin';
+import TerserPlugin from 'terser-webpack-plugin';
+import { BundleAnalyzerPlugin } from 'webpack-bundle-analyzer';
+
+import terserOptions from '../../scripts/buildtools/terser/config';
+import baseConfig from './webpack.common.config';
+
+import type { Configuration } from 'webpack';
+
+const prodConfig: Configuration = merge(
+  baseConfig,
+  {
+    mode: 'production',
+    devtool: 'source-map',
+    entry: {
+      default: {
+        import: [
+          // polyfill must appear first in entry point array
+          resolve(__dirname, '../..', './polyfill/modern.js'),
+          resolve(__dirname, 'src/index.ts'),
+        ],
+        filename: 'js/okta-sign-in.min.js',
```

Comment:
should `next` be included in the filename? or any other script handles that logic?

Reply:
Yeah good call, I'll update this along with the CSS filename for the release config